rcsb/db/define/SchemaDefBuild.py

```diff
@@ -765,30 +766,131 @@ def __createJsonLikeSchema(
+                                xD["properties"][schemaAttributeName] = {typeKey: "array", "items": atPropD}
+                                if emNesIterAttMd and "minItems" in emNesIterAttMd:
+                                    xD["properties"][schemaAttributeName]["minItems"] = emNesIterAttMd["minItems"]
+                                if emNesIterAttDesAlt:
+                                    xD["properties"][schemaAttributeName]["description"] = emNesIterAttDesAlt
+                            else:
+                                xD["properties"][schemaAttributeName] = atPropD
+
+                        # Populate nested child subcategory
+                        if xD["properties"]:
+                            if scnReqL:
+                                xD["required"] = scnReqL
+                            if scnHasUnitCard:
+                                scD["properties"][scNestedChild] = xD
+                            else:
+                                scD["properties"][scNestedChild] = {typeKey: "array", "items": xD}
+                                if scnMinUniqueItems and "minItems" in scnMinUniqueItems:
+                                    scD["properties"][scNestedChild]["minItems"] = scnMinUniqueItems["minItems"]
+                                if scnMinUniqueItems and "uniqueItems" in scnMinUniqueItems:
+                                    scD["properties"][scNestedChild]["uniqueItems"] = scnMinUniqueItems["uniqueItems"]
+                                if scnDesAlt:
+                                    scD["properties"][scNestedChild]["description"] = scnDesAlt
+
                     for atName in sorted(atNameList):
                         fD = aD[atName]
                         #
                         if subCategory not in [qtD["id"] for qtD in fD["SUB_CATEGORIES"]]:
                             continue
+
+                        # Skip attributes in nested child subcategories that have already been handled above
+                        if atName in scNesAttrL:
+                            continue
                         #
                         schemaAttributeName = convertNameF(atName)
                         if removeSubCategoryPrefix:
                             schemaAttributeName = schemaAttributeName.replace(subCategory + "_", "")
                         #
                         # isRequired = "mandatoryAttributes" in enforceOpts and fD["IS_MANDATORY"]
                         # JDW separate general support for mandatory attributes and support in subcategories.
-                        isRequired = mandatorySubcategoryAttributes and fD["IS_MANDATORY"]
+                        # isRequired = mandatorySubcategoryAttributes and fD["IS_MANDATORY"]
+                        isRequired = (
+                                mandatorySubcategoryAttributes and fD["IS_MANDATORY"]
+                                or ((catName, atName) in catAttrRequiredList)
+                            )
```

Comment:
@brindakv Can you please double-check/confirm if the parentheses are correct here for the boolean logic?